crates/pipeline-manager/src/db/storage_postgres.rs

```diff
@@ -28,6 +28,16 @@ use log::{debug, info, log, Level};
 use tokio_postgres::Row;
 use uuid::Uuid;
 
+/// Compute a hash of the UUID to match PostgreSQL's behavior.
```

Comment:
if you have to have code in rust for it anyways probably better to just use that rather than the complicated sql expression above if possible?

Reply:
as we want to limit the result from query to 1, it isn't possible to do it qithout sql expression.

Maybe instead of LIMIT 1 we just get all pipelines and keep checking next row until we get something?